tests/integration/upgrade-integration-test.sh

```diff
@@ -28,17 +29,20 @@ else
 fi
 
 echo 'Testing signed official docker image...'
-kubectl run dpod --image=docker.io/library/hello-world -lapp.kubernetes.io/instance=connaisseur >output.log 2>&1 || true
+kubectl run dpod-${RAND} --image=docker.io/library/hello-world -lapp.kubernetes.io/instance=connaisseur >output.log 2>&1 || true
 
-if [[ "$(cat output.log)" != 'pod/dpod created' ]]; then
+if [[ "$(cat output.log)" != "pod/dpod-${RAND} created" ]]; then
   echo 'Failed to allow signed image. Output:'
   cat output.log
   exit 1
 else
   echo 'Successfully allowed usage of signed image'
 fi
 
-kubectl delete pods --all || { echo 'Failed to delete test pods'; exit 1; }
+# Cleanup can be skipped in GitHub Actions, since uninstall is already tested in other integration test
```

Comment:
Should probably add the uninstall as well to do proper cleanup and make the comment apply

Reply:
You mean uninstall of Connaisseur? The integration test does not install Connaisseur, so I would not uninstall it as well

Reply:
Oh, I assumed it was more or less the same as the others, but then no install it is ;)